packages/lib-classifier/src/plugins/drawingTools/experimental/components/LineControls/LineControls.js

```diff
@@ -0,0 +1,168 @@
+  const STROKE_WIDTH = 1.5
+  const ARROW_STROKE_WIDTH = 0.5
+  const undoTransform = `translate(${position.get(activePosition).x + 15}, ${position.get(activePosition).y - 35}) scale(${1 / scale})`
+  const outerRadius = 40;
+  const innerRadius = 20;
+  
+  const StyledPath = styled('path')`
+    fill: ${FILL_COLOR};
+    &:hover {
+      cursor: pointer;
+      fill: ${HOVER_COLOR};
+    }
+  `
+
+  const TL = `M ${position.get(activePosition).x-outerRadius} ${position.get(activePosition).y}
+    A ${outerRadius} ${outerRadius} 0 0 1 ${position.get(activePosition).x} ${position.get(activePosition).y-outerRadius}
+    L ${position.get(activePosition).x} ${position.get(activePosition).y - innerRadius}
+    A ${innerRadius} ${innerRadius} 0 0 0 ${position.get(activePosition).x-innerRadius} ${position.get(activePosition).y}
+    Z`;
+
+  const TR = `M ${position.get(activePosition).x+outerRadius} ${position.get(activePosition).y}
+    A ${outerRadius} ${outerRadius} 0 0 0 ${position.get(activePosition).x} ${position.get(activePosition).y-outerRadius}
+    L ${position.get(activePosition).x} ${position.get(activePosition).y - innerRadius}
+    A ${innerRadius} ${innerRadius} 0 0 1 ${position.get(activePosition).x+innerRadius} ${position.get(activePosition).y}
+    Z`;
+
+  const BL = `M ${position.get(activePosition).x-outerRadius} ${position.get(activePosition).y}
+    A ${outerRadius} ${outerRadius} 0 0 0 ${position.get(activePosition).x} ${position.get(activePosition).y+outerRadius}
+    L ${position.get(activePosition).x} ${position.get(activePosition).y + innerRadius}
+    A ${innerRadius} ${innerRadius} 0 0 1 ${position.get(activePosition).x-innerRadius} ${position.get(activePosition).y}
+    Z`;
+
+  const BR = `M ${position.get(activePosition).x+outerRadius} ${position.get(activePosition).y}
+    A ${outerRadius} ${outerRadius} 0 0 1 ${position.get(activePosition).x} ${position.get(activePosition).y+outerRadius}
+    L ${position.get(activePosition).x} ${position.get(activePosition).y + innerRadius}
+    A ${innerRadius} ${innerRadius} 0 0 0 ${position.get(activePosition).x+innerRadius} ${position.get(activePosition).y}
+    Z`;
+
+  const C = `M ${position.get(activePosition).x-innerRadius} ${position.get(activePosition).y}
+    A ${innerRadius} ${innerRadius} 0 0 0 ${position.get(activePosition).x+innerRadius} ${position.get(activePosition).y}
+    A ${innerRadius} ${innerRadius} 0 0 0 ${position.get(activePosition).x-innerRadius} ${position.get(activePosition).y}
+    Z`;
+
+  function move(e) {
+    if (activePosition == 'tl') {
+      setActivePosition('tr');
+    } else {
+      setActivePosition('tl');
+    }
+  }
+
+  function destroy() {
+    onDelete();
+  }
+
+  return (
+    <g
+      ref={ref}
+      x={position.get(activePosition).x}
+      y={position.get(activePosition).y}
+      role='button'
+      aria-label={ARIA_LABEL}
+      focusable='true'
+      stroke={STROKE_COLOR}
+      strokeWidth={STROKE_WIDTH}
+    >
```

Comment:
The whole component is marked up as a focusable button here, but it's actually made up of four different buttons. If I tab to this, there's a button called 'Line Controls' in the page focus order, which does nothing when I focus it and press Space or Enter.

Reply:
The [WebAIM guide to keyboard testing](https://webaim.org/techniques/keyboard/#testing)  is linked from the PR checklist, but worth repeating here. You should check your work from the keyboard, to make sure that buttons and clickable elements are accessible without using a mouse or trackpad.

Line controls here needs to be updated to make each clickable area into an actual button, with button states and roles. It might actually be easier to refactor the line controls to use HTML `<button>`  elements in a toolbar menu instead of a clickable SVG graphic.

Reply:
On a touch device, there's no visible label for each of the four buttons.

Reply:
I've been trying this out on Etch-A-Cell here, both in desktop Firefox and on iOS:
https://fe-project-branch.preview.zooniverse.org/projects/h-spiers/etch-a-cell/classify/workflow/8921

On desktop, I do the following:
1. Trace a segment of the cell wall.
1. Zoom in with + and pan with the cursor keys to a section that I want to redraw.

What I'm finding is that the Line Controls component vanishes when I zoom in, so I can't undo/redo unless I  zoom back out with -. It's really difficult, maybe even impossible, to use with a real cell segmentation task.

Reply:
Spent a good chunk of time on this over the past week and I've been able to fix this to the best of my understanding of Accessibility guidelines. Specifically, the Line Controls now stay in view regardless of zoom. Pressing Enter or Spacebar on the button when keyed into focus performs the action.

Reply:
The new code looks a lot better. Tab indexes need changing and the buttons need icons or some visual indication of what they do. The tooltips are only available on hover, which means they won’t show on touch devices.

Reply:
The tooltip labels don't show when the SVG buttons are focused from the keyboard.

<img width="278" alt="The control wheel with one button focused. There's no visible label for the focused control." src="https://user-images.githubusercontent.com/59547/226856941-e377280c-fb6d-4bb9-af26-7986c32b63e6.png">

Reply:
Compare that with the tooltip labels on the SVG lines for a transcription subject, when you tab through the page.
https://www.zooniverse.org/projects/pmlogan/poets-and-lovers/classify/workflow/21362/subject-set/104806
<img width="410" alt="A page from Poets and Lovers. One SVG line is focused, with a visible label." src="https://user-images.githubusercontent.com/59547/226857567-6f24a590-e410-4632-b116-f63bb10367a2.png">